src/Command/Argument/Search/CreateArguments.php

```diff
@@ -0,0 +1,248 @@
+<?php
+
+/*
+ * This file is part of the Predis package.
```

Comment:
IMHO new code, stop adding the copyright...

Reply:
Since, one of latest @tillkruss PR, copyright becomes mandatory linter option, it's automatically added by IDE according to linter settings, so it will break CI from now

Reply:
@tillkruss WDYT?

Reply:
Let's use a dedicated copyright for all files, not individual headers for various files.